pkg/ru_fa/reader.go

```diff
@@ -0,0 +1,292 @@
+package ru_fa
+
+import (
+	"fmt"
+	"io"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/moov-io/watchman/pkg/download"
+
+	"github.com/PuerkitoBio/goquery"
+	"github.com/mozillazg/go-unidecode"
+)
+
+// ---------- Constants and Helper Functions ----------
+
+const (
+	urlToDownload = "https://ru.ruwiki.ru/wiki/%D0%A1%D0%BF%D0%B8%D1%81%D0%BE%D0%BA_%D0%B8%D0%BD%D0%BE%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%BD%D1%8B%D1%85_%D0%B0%D0%B3%D0%B5%D0%BD%D1%82%D0%BE%D0%B2_(%D0%A0%D0%BE%D1%81%D1%81%D0%B8%D1%8F)"
```

Comment:
Is this the simplest URL we can use for this page? That doesn't look like a stable URL.

Reply:
Yes it is. It's just  the page name in urlencoded unicode.